js/util.js

```diff
@@ -22,7 +22,7 @@ const getRandomElementsArray = (randomElementsList) => {
   return _.sampleSize(randomElementsList, quantityElements);
 };
 
-const getRandomElement = (elementsArray) => {
+const getRandomElement = ([elementsArray]) => {
   return _.sample([elementsArray]);
```

Comment:
Это неправильное исправление. Правильно будет так
```suggestion
const getRandomElement = (elementsArray) => {
  return _.sample(elementsArray);
```

Reply:
Я тут просто изменил, так как почему то без явного указания, что будет массив, у меня функция возвращала все элементы массива, а не один элемент почему-то.